stdlib/public/core/CMakeLists.txt

```diff
@@ -121,7 +120,6 @@ set(SWIFTLIB_SOURCES
   ### PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER ###
   Availability.swift
   Bit.swift
-  CollectionMirrors.swift.gyb
   CollectionOfOne.swift
   ExistentialCollection.swift.gyb
   Mirror.swift
```

Comment:
I think it makes sense to make as much progress as possible in the incremental development spirit.  If we can't remove something without redesigning the runtime, it shouldn't block progress where we can make it.